content/docs/buildpack-author-guide/create-buildpack/building-blocks-cnb.md

```diff
@@ -23,7 +23,7 @@ Create the `ruby-buildpack/buildpack.toml` file and copy the following into it:
 <!-- test:file=ruby-buildpack/buildpack.toml -->
 ```toml
 # Buildpack API version
-api = "0.2"
+api = "0.5"
```

Comment:
Moved to API v0.5. The biggest change in this PR is the move from build plan `requires.version` to `requires.metadata.version`